### ISSUE TITLE
Menu does not work anymore after update to 2.2.3

### ISSUE THREAD

**Issue by robru123** [turnkeymobileapps/help-center#51]
Apple iPhone 6 iOS 10.3.1

![20170418_124747000_ios](https://cloud.githubusercontent.com/assets/24276544/25131794/5b129cce-2447-11e7-8448-fed765e86006.png)

**Comment by turnkeymobileapps** [comment]
Can you please share a screenshot of the "Turnkey Mobile Apps" style properties for this style?

**Comment by robru123** [comment]
![menu](https://cloud.githubusercontent.com/assets/24276544/25133759/f283b740-244d-11e7-8d47-a02436ee346a.jpg)

**Comment by turnkeymobileapps** [comment]
Check the auto-hide checkbox; does that resolve it?

**Comment by vbresults** [comment]
Also, change navigation expression to:
```javascript
SkinMyForum.mobileNav()
`

**Comment by turnkeymobileapps** [comment]
Check the "Navigation: Auto-hide" setting's checkbox, then, change the "Toggle Script Expression" to:
```javascript
SkinMyForum.mobileNav()
`
These are supposed to be defaults for Twenty Two and I'm not sure how they ended up like this; were they manually changed?

**Comment by robru123** [comment]
> Check the auto-hide checkbox; does that resolve it?
Done but the problem remains the same

**Comment by turnkeymobileapps** [comment]
I revised the post with new instructions

**Comment by robru123** [comment]
Change to: 
SkinMyForum.mobileNav() `
or
SkinMyForum.mobileNav()

**Comment by vbresults** [comment]
The second one without the tick and space

**Comment by turnkeymobileapps** [comment]
The second one without the tick and space

**Comment by robru123** [comment]
I used the second one: SkinMyForum.mobileNav()
Now when you open the app, opens the menu first!

![20170418_140712000_ios](https://cloud.githubusercontent.com/assets/24276544/25135076/920ed8aa-2451-11e7-8980-d122b9d24d9b.png)

**Comment by robru123** [comment]
I have to go now, I will continue to test later in the day ;)

**Comment by robru123** [comment]
Is there a possibility that the menu works.
But does not open automatically when you start the app?

**Comment by turnkeymobileapps** [comment]
It shouldn't open it automatically; but there were bugs that were just fixed in both the add-on and style minutes ago -- please download the zip and overwrite the files, add-on and twenty two style again and let me know whether or not the issue persists

**Comment by robru123** [comment]
Thank you for this update, I try it tonight :)

**Comment by robru123** [comment]
The problem is solved with this new update version 2.2.3a, my thanks for this 👍 
I'm curious to solve Shield Loading Pages this week.

**Comment by ulrihovich** [comment]
ap version 2.2.4
App ID: com.aquapech.forum.app
ANDROID
NEXUS 5X
Menu buttone closes logo and menu don't  appear
![screenshot_20170514-003233](https://cloud.githubusercontent.com/assets/28564085/26029444/cb8e772a-383d-11e7-9806-1cdbb088904a.png)
![screenshot_20170514-003239](https://cloud.githubusercontent.com/assets/28564085/26029445/cb9016de-383d-11e7-8b61-e364d1a39c82.png)
